services/121-service/src/events/events.service.spec.ts

```diff
@@ -192,7 +192,7 @@ describe('EventsService', () => {
       chunk: 2000,
     });
   });
-
+  // I don`t know how to fix this.
```

Comment:
We can have a look together

Reply:
If you have today, you can write me and check this.

Reply:
If it's ok, I'd say to merge this like this:
```suggestion
  // XXX: I don`t know how to fix this.
```

So we can find it again, and then we come back to it once I'm back from holidays.

Reply:
okay, I will add comments on everything that is not easy to fix.